lib/codegen/selection/generator.cc

```diff
@@ -441,18 +441,18 @@ std::tuple<Value*, Value*, Value*, Value*> generator::fp8x4_to_fp16x4(Value *in0
   "lop3.b32 $1, b1, 0x80008000, a1, 0xf8; \n\t"
   "}", "=r,=r,r", false);
   Value *packed_in = UndefValue::get(vec_ty(i8_ty, 4));
-  packed_in = insert_elt(packed_in, in0, (int)0);
-  packed_in = insert_elt(packed_in, in1, (int)1);
-  packed_in = insert_elt(packed_in, in2, (int)2);
-  packed_in = insert_elt(packed_in, in3, (int)3);
+  packed_in = insert_elt(packed_in, in0, (uint64_t)0);
+  packed_in = insert_elt(packed_in, in1, (uint64_t)1);
+  packed_in = insert_elt(packed_in, in2, (uint64_t)2);
+  packed_in = insert_elt(packed_in, in3, (uint64_t)3);
```

Comment:
@ptillet can you clarify if the perf test failed due to improvement or regression? :-D

I am wondering if this change might be the cause for the regression test to fail

Reply:
Interestingly, without this change the project does not build even on Linux for me due to inability to choose the right overload of `CreateInsertElement`. o-O

Reply:
Unfortunately it appears to be a regression :O It surprises me because I do not see how this could be the case, but at the same time I have never seen any false positive in the regression test yet. Let me rerun it and see if it fails again.

Reply:
I think it was a false positive. Reran it twice and didn't see a regression. Merging this PR

Reply:
@lostmsu actually is it possible to rebase `master` so that this PR is not out-of-date anymore? I will merge asap after this PR is up-to-date

Reply:
@ptillet rebased it
BTW, GitHub has an option to do so when there are not conflicts.